EIPS/eip-5732.md

```diff
@@ -134,6 +136,7 @@ contract BarVote is ISimpleVote, IERC_COMMIT {
 
     function commit(bytes32 _commitment, bytes calldata _extraData) {
         lastCommits[msg.sender] = _commitment;
+        emit Commit(...);
     }
 
     function vote(address _proposalId, uint8 _optionId, bytes calldata _extraData) {
```

Comment:
On L150, the keccak arg shouldn't be `_amount` used. Instead, this line should be:

```Solidity
return keccak256(abi.encodePack(_sender, _proposalId, _optionId, _secretSalt);
```

Reply:
Good catch, thank you!

Reply:
thank you matt. If you are interested, i love to invite you to  create a full featured ref impl in github.com/ercref/contracts let me know if you have any question at https://discord.gg/ZvAPfsNr